src/shared/ui/survey/GeolocationItem/index.ts

```diff
@@ -0,0 +1 @@
+export { default as GeolocationItem } from './GeolocationItem';
```

Comment:
Please move the GeolocationItem.tsx out of the GeolocationItem folder since it is only one file